api/tests/integration/infrastructure/repositories/organization-learner-repository_test.js

```diff
@@ -2297,6 +2327,36 @@ describe('Integration | Infrastructure | Repository | organization-learner-repos
       expect(result).to.deep.equal([]);
     });
 
+    it('should not return an organization learner not reconciliated', async function () {
+      // given
+      const { organizationId } = databaseBuilder.factory.buildOrganizationLearner({ userId: null });
+
+      databaseBuilder.factory.buildOrganizationFeature({ featureId, organizationId });
+      await databaseBuilder.commit();
+
+      // when
+      const result = await organizationLearnerRepository.findByOrganizationsWhichNeedToComputeCertificability();
+
+      // then
+      expect(result).to.deep.equal([]);
```

Comment:
On peut aussi faire:
```suggestion
      expect(result).to.be.empty;
```